src/components/ebay-tab/migrator.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function migrator(el, context) {
+    context.deprecate('<ebay-tab> has been renamed to <ebay-tabs>');
+    const oldAttribute = 'on-tab-select';
+    const newAttribute = 'on-tabs-select';
+    el.setTagName('ebay-tabs');
+    if (el.hasAttribute(oldAttribute)) {
+        const attribute = el.getAttribute(oldAttribute);
+        el.removeAttribute(oldAttribute);
+        attribute.name = newAttribute;
+        el.addAttribute(attribute);
+    }
+
+    el.forEachChild(child => {
+        if (child.tagName === 'ebay-tab-heading') {
```

Comment:
This is going to cause issues if either of these are in a `<for>` or `<if>`. We should use traverse here.